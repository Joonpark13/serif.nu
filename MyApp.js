import React from 'react';

function MyApp() {
  return <div>My first Serif.nu ticket</div>;
}

export default MyApp;




