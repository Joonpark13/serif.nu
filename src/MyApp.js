import React from 'react';

function MyApp() {
  return <div>I love web development.</div>;
}

export default MyApp;
