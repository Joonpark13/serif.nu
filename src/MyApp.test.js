import React from 'react';
import { shallow } from 'enzyme';
import MyApp from './MyApp';

describe('MyApp', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<MyApp />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
