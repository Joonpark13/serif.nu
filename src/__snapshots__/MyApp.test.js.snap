// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp renders correctly 1`] = `
<div>
  I love web development.
</div>
`;
